lib/zendesk_api/collection.rb

```diff
@@ -356,30 +356,27 @@ def intentional_obp_request?
       Helpers.present?(@options["page"]) && !cbp_request?
     end
 
-    def should_try_cbp?(path_query_link)
-      not_supported_endpoints = %w[show_many]
-      not_supported_endpoints.none? { |endpoint| path_query_link.end_with?(endpoint) }
+    def supports_cursor_based_pagination?
```

Comment:
This is good.

Minor: Considering we're using cbp and obp freely, I wonder if `supports_cbp?` would be better.